### ISSUE TITLE
Remove redundant documentation comments

### ISSUE THREAD

**Issue by Rangi42** [pret/pokecrystal#741]
Some values in WRAM have comments that reiterate what's already described with their associated constants or data.

In particular, the `wPlayerSubStatus*` labels have long comments to remove. (The `SUBSTATUS_*` constants should also be renamed `SUBSTATUS1_*`, `SUBSTATUS2_*`, etc, to make it clear which of the five substatus bytes they correspond to.)

**Comment by Rangi42** [comment]
The substatus comments are removed; there are still others, mostly for certain bitfields.

**Comment by coltongit** [comment]
What 'certain bitfields' do you mean exactly? I'm kinda confused

**Comment by mid-kid** [comment]
No, she means the comments explaining what each bit in a wram address means. Those are redundant when they're already described as constants.

**Comment by coltongit** [comment]
> No, she means the comments explaining what each bit in a wram address means. Those are redundant when they're already described as constants.

That makes a lot more sense, thank you for the clarification
